### PR TITLE
Add version_added for all new features

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Adds `version_added: '1.0.0'` to all new features. I'll stick to `1.0.0` until we decided how we'll version and what's the next release.

CC @gundalow @abadger @Andersson007 @Akasurde 

##### ISSUE TYPE
- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request

##### COMPONENT NAME
multiple

### CODE REVIEW

**Comment by felixfontein** [comment]
I've prepended a commit which adds `version_added: '1.0.0'` for all features added after stable-2.9 was branched from ansible/ansible, and before content was moved to collections. This is necessary for having a proper changelog for the ACD 2.10 release which contains all new features, and not just the ones added after the collection split.

(The exact value for `version_added` is debatable, but modifying the commit is a lot easier than re-creating them from commit history another time.)

File: plugins/modules/cloud/alicloud/ali_instance.py [hunk 1]
```diff
@@ -106,6 +106,7 @@
           The sequential suffix ranges from 001 to 999.
       default: False
       type: bool
+      version_added: '1.0.0'
```
[the remark this answers is not in the document]
Comment (felixfontein) [reply plugins/modules/cloud/alicloud/ali_instance.py:109]: I'm not sure whether Galaxy versions are always three numbers, or can also be just two. If they are always three numbers (x.y.z), then I think we should use x.y.z here as well to indicate the exact version where the feature was first introduced.

**Comment by felixfontein** [comment]
Updated for latest commit f340b39bb9f80e7121fbd864c0d82581a7bfac1f

File: plugins/modules/cloud/alicloud/ali_instance.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Andersson007) [reply plugins/modules/cloud/alicloud/ali_instance.py:109]: ok

**Comment by felixfontein** [comment]
Updated for last commit `43c805f7db1b6764e48c3ac3ba995040e967018f`

**Comment by felixfontein** [comment]
Updated for last commit 196d3205cab45473e75867e1243a1009af878607

**Comment by felixfontein** [comment]
Updated for last commit a0a534508a437e1ba5f27929df619b3fcf19403c.

**Comment by felixfontein** [comment]
Updated for last commit 599fc7f17bcd876905311ef01a901a00210cbb43.

**Comment by AlanCoding** [comment]
how will `version_added` for old features be handled? I'm a little confused on that.

**Comment by felixfontein** [comment]
It won't. Old features were already there in 2.9, so they are not new for community.general, so they don't get/keep a `version_added`.

**Comment by felixfontein** [comment]
Updated for last commit a0c5580fc3887b236e008f626e83521ae6c1630f.

**Comment by felixfontein** [comment]
Updated for last commit 647abcb390cf1077d744586beb1ea8fde5ba09c5.

**Comment by felixfontein** [comment]
Updated for last commit 2aa84f07f11dfe2cfdfa22ef2f4da09d71ae2a1b.

**Comment by felixfontein** [comment]
epel still has problems. Since this change should not affect any of the tests, merging. (If it breaks something, I'll fix it later.)

**Pull request merged**